### PR TITLE
bears: Add npm requirements

### PR DESCRIPTION
**Pull request by Adrianzatreanu**

### CODE REVIEW

**Comment by Adrianzatreanu** [comment]
would like extra care reviewing the ``CSSAutoPrefixBear.py``, as im not 100% sure thats perfect

File: bears/coffee_script/CoffeeLintBear.py [hunk 1]
```diff
@@ -14,6 +15,7 @@ class CoffeeLintBear:
     """
 
     LANGUAGES = {"CoffeeScript"}
+    REQUIREMENTS = NpmRequirement('coffeelint', '1')
```
Comment (sils1297) [inline bears/coffee_script/CoffeeLintBear.py]: set please

**Comment by sils1297** [comment]
ack https://github.com/coala-analyzer/coala-bears/pull/533/commits/c526bba6d419528e0907ae5fd76f1eec3262ce77

**Comment by sils1297** [comment]
ack https://github.com/coala-analyzer/coala-bears/pull/533/commits/9e7f856e618ff96fc8cce5e3030e0f217778519f https://github.com/coala-analyzer/coala-bears/pull/533/commits/2c45d959e89dee9a6bca8c879c6ce3dcafc593c2

**Comment by sils1297** [comment]
ACK https://github.com/coala-analyzer/coala-bears/pull/533/commits/18382ed9504ebb0d4fbee4ad684ccda771ac8cde

**Comment by Adrianzatreanu** [comment]
reack @Adrianzatreanu	bears: Add npm requirements			b82d84a
 @Adrianzatreanu	bears: Change PythonRequirement to PipRequirement  …			1ddaf6c
 @Adrianzatreanu	bears: Add gem requirements			82e2272

**Comment by Adrianzatreanu** [comment]
reack 82e2272

**Comment by Adrianzatreanu** [comment]
reack 1ddaf6c

**Comment by Adrianzatreanu** [comment]
reack b82d84a

**Comment by Adrianzatreanu** [comment]
@rultor merge

**Pull request merged**